config/database.github.yml

```diff
@@ -0,0 +1,23 @@
+# MySQL Configuration used for Github Actions
+# https://alexrs.me/2021/github-actions-mysql-rails
+development:
+  adapter: mysql2
+  database: autolab_development
+  pool: 5
+  username: root
+  password: root
+  socket: /var/run/mysqld/mysqld.sock
+  host: localhost
+  variables:
+    sql_mode: NO_ENGINE_SUBSTITUTION
+
+test:
+  adapter: mysql2
```

Comment:
How are you able to use MySQL even without setting that up in GitHub actions?

Reply:
MySQL is preinstalled, and I started the service with `sudo /etc/init.d/mysql start`

Links:
https://ovirium.com/blog/how-to-make-mysql-work-in-your-github-actions/
https://github.blog/changelog/2020-02-21-github-actions-breaking-change-ubuntu-virtual-environments-will-no-longer-start-the-mysql-service-automatically/